XamMacSwiftUITest.FSharp/AppDelegate.fs

```diff
@@ -10,11 +10,11 @@ open SwiftUI
 [<Register("AppDelegate")>]
 type AppDelegate() =
     inherit NSApplicationDelegate()
-    override self.ApplicationShouldTerminateAfterLastWindowClosed(_) = GC.Collect(); false
+    override self.ApplicationShouldTerminateAfterLastWindowClosed(_) = GC.Collect(); true
     override self.DidFinishLaunching(_) =
         let flags = NSWindowStyle.Titled ||| NSWindowStyle.Closable ||| NSWindowStyle.Miniaturizable ||| NSWindowStyle.Resizable ||| NSWindowStyle.FullSizeContentView
+
         let window = new NSWindow(CGRect(0., 0., 480., 300.), flags, NSBackingStore.Buffered, false)
-        window.WillClose.Add(fun _ -> window.ContentView <- NSTextField.CreateLabel("CLOSING"); GC.Collect())
```

Comment:
This appears to be the culprit of the crash on exit.

Reply:
Yep, I added this code to force the objects to be released by native code and then collected by the GC so we could debug the object lifecycles (and clearly there's a bug)